charter.html

```diff
@@ -436,6 +448,17 @@ <h2>
         </p>
 
 
+      <section id="datapolicy">
+        <h2>
+          Data Policy
+        </h2>
+        <p>
+          The W3C patent policy addresses the need for those that have patents over essential inputs to license them on FRAND terms to all Members. Inputs that may not be protected by patents but may be needed by others for implementation of a specification or standard also need to be available to all on FRAND terms to avoid the same issue of inputs being available to some and not others from arising.
```

Comment:
Please define FRAND before first use by spelling it out?

Reply:
Ahead of a further commit it means Fair Reasonable And Non Discriminatory.

Reply:
Ok. I'm still unclear on what the intent of this is? Is it for access to test data? Is it to try and require something to be included in the specifications? What is it intended to protect access to?